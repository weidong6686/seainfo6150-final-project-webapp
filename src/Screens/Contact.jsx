import React from 'react'; 
import './css/About.css';
const Contact = (props) => {
    return (
        <div>
            <img className="img-contact" src="https://i.ibb.co/LJXJnQC/Contact.png" alt="ContactUs" />
            <div className="content">
                <h2>Phone:</h2>
                <p>Phone call at +1 620 616 6610 (Monday to Sunday - 8.00am until 10:00pm) including Public Holidays.</p>
                
                <h2>Email:</h2>
                <p>Email to <b>shopeeservice@shopee.com</b>, we will reply you in 24 hours.</p>
            </div>
        </div>
    )
}

export default Contact