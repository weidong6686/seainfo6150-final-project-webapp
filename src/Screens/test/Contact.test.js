import React from "react";
import Contact from "../Contact.jsx";

describe("Contact tests", () => {
  it("renders correctly", () => {
    const { container } = (<Contact />);
    expect(container).toMatchSnapshot();
  });
});