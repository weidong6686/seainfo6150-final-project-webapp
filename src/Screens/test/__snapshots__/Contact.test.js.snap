// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact tests renders correctly 1`] = `undefined`;
